test/jasmine/assets/mouse_event.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(type, x, y, opts) {
         ev;
 
     if(type === 'scroll') {
-        ev = new window.WheelEvent('wheel', opts);
+        ev = new window.WheelEvent('wheel', Object.assign({}, fullOpts, opts));
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1522/commits/745b9535a06c1c850013a99504c53912f5b30700

---

I should really be reviewing all commits before making comments.